build/common/installer/scripts/tomlparser.rb

```diff
@@ -161,6 +162,26 @@ def populateSettingValuesFromConfigMap(parsedConfig)
           puts "config:: WARN: container logs V2 is disabled and is required for multiline logging. Disabling multiline logging"
           @logEnableMultiline = "false"
         end
+
+        multilineLanguages = parsedConfig[:log_collection_settings][:enable_multiline_logs][:stacktrace_languages]
```

Comment:
add check for parsedConfig[:log_collection_settings][:enable_multiline_logs][:stacktrace_languages]

Reply:
its added on line 167